Resources/public/js/views/ez-confirmboxview.js

```diff
@@ -178,6 +224,29 @@ YUI.add('ez-confirmboxview', function (Y) {
             },
 
             /**
+             * View of the confirmBox.
+             * A Y.View that can be provided to be displayed in the confirm box
+             *
+             * @attribute view
+             * @default {}
+             * @type {Object}
```

Comment:
actually it stores a Y.View (or null given https://github.com/ezsystems/PlatformUIBundle/pull/658/files#r72792835)